openc3-cosmos-init/plugins/packages/openc3-vue-common/src/components/Graph.vue

```diff
@@ -1507,8 +1510,19 @@ export default {
         } else {
           let index = bs(this.data[0], time, this.bs_comparator)
           if (index >= 0) {
-            // Found the slot in the existing data
-            this.set_data_at_index(index, time, data[i])
+            // Found a slot with the exact same time value
+            // Handle duplicate time by subtracting a small amount until we find an open slot
+            while (index >= 0) {
+              time -= 1e-6 // Subtract a microsecond
```

Comment:
1us is below the precision of a float64 with unix epoch.   Probably needs to be 10us to guarantee a unique floating point number.   Does it need to be unique, or just exist in the array?